tests/sles4sap/publiccloud/hana_sr_takeover.pm

```diff
@@ -35,7 +35,7 @@ sub run {
     $self->{my_instance} = $target_site;
 
     # Check initial cluster status
-    $self->run_cmd(cmd => 'zypper -n in ClusterTools2');
+    $self->run_cmd(cmd => 'zypper -n in ClusterTools2', timeout => 300);
```

Comment:
is it related to full update? Can this be moved in a dedicated PR?

Reply:
@lilyeyes perhaps what you can do is move the change into this file to its own commit so it can easily reverted if needed.

Reply:
Done, submitted another commit in this PR.